wimm.Missionary/IConversionMap.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace wimm.Missionary
+{
+    /// <summary>
+    /// A map of types to conversions. The map's type is the convert-from type for all conversions
+    /// in the map, and the keys are the convert-to types for their corresponding values.
+    /// </summary>
+    /// <typeparam name="T">The convert-from type.</typeparam>
+    public interface IConversionMap<T>
+    {
+        // TODO:TS Consider using a maybe instead of null
+
+        /// <summary>
+        /// Gets the conversion from <typeparamref name="T"/> to <typeparamref name="U"/>.
+        /// </summary>
+        /// <typeparam name="U">The convert-to type.</typeparam>
+        /// <returns>
+        /// The conversion for <typeparamref name="U"/>, or <c>null</c> if the map does not contain
+        /// a conversion for the specified type.
+        /// </returns>
+        IConversion<T, U> Get<U>();
+
+        // TODO: Consider throwing if the same typed conversion is set more than once.
```

Comment:
I think this is the only reasonable way to do it. I can't imagine why you would need to change this conversion at runtime. It would pooch a lot of the guarantees that you could otherwise make re:construction

Reply:
wf

Reply:
:ship: it?

Reply:
oops left the TODO, wf again